ko/lessons/basics/sigils.md

```diff
@@ -48,11 +48,11 @@ iex> ~C/2 + 7 = #{2 + 7}/
 '2 + 7 = #{2 + 7}'
 ```
 
-We can see the lowercased `~c` interpolates the calculation, whereas the uppercased `~C` sigil does not. We will see that this uppercase / lowercase sequence is a common theme throughout the built in sigils.
+소문자 `~c`는 수식을 계산하여 문자 리스트에 확장하지만, 대문자 시길 `~C`는 그렇지 않음을 알 수 있습니다. 여러분은 이러한 소문자 / 대문자 구조가 모든 내장 시길에 걸쳐 주를 이룸을 알게 될 것입니다.
```

Comment:
그쪽이 더 좋을 것 같네요.


    _____________________________
From: Dalgona. <notifications@github.com>
Sent: 화요일, 5월 24, 2016 3:17 오후
Subject: Re: [Dalgona/elixir-school] [KO] Translation of Sigils Lesson (#4)
To: Dalgona/elixir-school <elixir-school@noreply.github.com>
Cc: Yi, Sang-won Leo <public@leo.re.kr>, Mention <mention@noreply.github.com>




In ko/lessons/basics/sigils.md:> @@ -48,11 +48,11 @@ iex> ~C/2 + 7 = #{2 + 7}/>  '2 + 7 = #{2 + 7}'>  ```>  > -We can see the lowercased `~c` interpolates the calculation, whereas the uppercased `~C` sigil does not. We will see that this uppercase / lowercase sequence is a common theme throughout the built in sigils.> +소문자 `~c`는 수식을 계산하여 문자 리스트에 확장하지만, 대문자 시길 `~C`는 그렇지 않음을 알 수 있습니다. 여러분은 이러한 소문자 / 대문자 구조가 모든 내장 시길에 걸쳐 주를 이룸을 알게 될 것입니다.

@devleoper 다만 "구별할 수 있습니다" 대신에 "구별할 수 있을 것입니다"라고 쓰는 편이 더 괜찮을 것 같은데요

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub